src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -872,13 +869,15 @@ void Chttp2ServerListener::OnAccept(void* arg, grpc_endpoint* tcp,
       // This ref needs to be taken in the critical region after having made
```

Comment:
I think this comment needs to be updated a bit.  It seems like it originally referred only to the ref on the listener, but it should now refer to both that and the ref on the TCP server.